scalikejdbc-core/src/main/scala/scalikejdbc/DBConnectionAttributesWiredResultSet.scala

```diff
@@ -8,30 +8,37 @@ import java.util
 import java.util.Calendar
 
 /**
- * TimeZone conversion proxy of ResultSet
+ * ResultSet proxy which holds current DBConnectionAttributes.
  */
-case class ResultSetProxy(underlying: ResultSet, connectionAttributes: DBConnectionAttributes) extends ResultSet {
+private[scalikejdbc] class DBConnectionAttributesWiredResultSet(
+    underlying: ResultSet,
+    connAttributes: DBConnectionAttributes) extends ResultSet {
 
-  private[this] def convertTimeZoneIfEnabled(timestamp: Timestamp): Timestamp =
-    if (connectionAttributes.timeZoneSettings.conversionEnabled) {
-      val serverTimeZone = connectionAttributes.timeZoneSettings.serverTimeZone
+  /**
+   * Converts Timestamp value to an appropriate timezone.
+   */
+  private[this] def convertTimeZoneIfNeeded(timestamp: Timestamp): Timestamp =
+    if (connAttributes.timeZoneSettings.conversionEnabled) {
+      val serverTimeZone = connAttributes.timeZoneSettings.serverTimeZone
       val clientTimeZone = java.util.TimeZone.getDefault
       TimeZoneConverter.from(serverTimeZone).to(clientTimeZone).convert(timestamp)
     } else {
       timestamp
     }
 
-  /**
```

Comment:
This comment will be shown as method comment for the `getTimestamp(Int)` in scaladoc. You should avoid using comments which start with double asterisk here.